src/TextEditor/TextEditor.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
What about displaying Markdown or formulas? Might be nice to show in the storybook and not too hard to add.